--- x264-0.svn20070613/Makefile	2007-06-13 15:52:11.000000000 +0200
+++ x264-0.svn20070613/Makefile	2007-06-13 15:52:41.000000000 +0200
@@ -79,7 +79,8 @@
 $(SONAME): .depend $(OBJS) $(OBJASM)
 	$(CC) -shared -o $@ $(OBJS) $(OBJASM) -Wl,-soname,$(SONAME) $(LDFLAGS)
 
-x264$(EXE): $(OBJCLI) libx264.a 
+x264$(EXE): $(SONAME)
+x264$(EXE): $(OBJCLI)
 	$(CC) -o $@ $+ $(LDFLAGS)
 
 libx264gtk.a: muxers.o libx264.a

